docs/source/api/user-guide/polars/expressions/manipulation.ipynb

```diff
@@ -435,6 +467,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "If the fill expression is not broadcast-able (references other columns), \n",
```

Comment:
If we don't rely on the notion of "broadcasting" down the road, could we just say "If the fill expression does not reference other columns ..."?  It feels like we're introducing jargon that isn't necessary.